File: pinyougou-parent/pinyougou-manager-controller/src/main/webapp/js/service/brandService.js
//服务层
app.service('brandService',function($http){
	    	
	//查询所有商品品牌
	this.findAll=function(){
		return $http.get('../brand/findAll.do');		
	}
	//分页查询商品品牌
	this.findPage=function(page,rows){
		return $http.get('../brand/findPage.do?page='+page+'&rows='+rows);
	}
	//查询指定的商品品牌
	this.findOne=function(id){
		return $http.get('../brand/findBrand.do?id='+id);
	}
	//增加商品品牌
	this.add=function(entity){
		return  $http.post('../brand/add.do',entity );
	}
	//修改商品品牌
	this.update=function(entity){
		return  $http.post('../brand/update.do',entity );
	}
	//删除商品品牌
	this.dele=function(ids){
		return $http.get('../brand/delete.do?ids='+ids);
	}
	//搜索商品品牌
	this.search=function(page,rows,searchEntity){
		return $http.post('../brand/search.do?page='+page+"&size="+rows, searchEntity);
	}    
	//下拉列表数据
	this.selectOptionList=function(){
		return $http.get('../brand/selectOptionList.do');
	}
	
});

File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.pinyougou.entity.PageResult;
import com.pinyougou.pojo.TbBrand;

/**
 * 
 * Title:BrandService.java
 * Description: 品牌接口
 * @author xll
 * @date 2019年1月25日 下午6:33:26
 * @version 1.0
 *
 */
public interface BrandService {

	//查询所有品牌
	public List<TbBrand> findAll();
	
	/**
	 * 
	 * Title:findPage
	 * Description: 分页品牌查询
	 * @param pageNum 当前页数
	 * @param pageSize 每页记录数
	 * @return 分页对象
	 */
	public PageResult findPage(int pageNum,int pageSize);
	
	/**
	 * 增加商品品牌
	 */
	public void addBrand(TbBrand brand);
	
	/**
	 * Title:findBrandById
	 * Description:根据id查询商品品牌信息
	 * @param id 商品品牌id
	 * @return 商品品牌对象
	 */
	public TbBrand findBrandById(Long id);

	/**
	 * Title:updateBrand
	 * Description: 修改商品品牌信息
	 * @param brand 商品品牌对象
	 */
	public void updateBrand(TbBrand brand);
	
	/**
	 * Title:delete
	 * Description: 根据id删除商品品牌
	 * @param ids
	 */
	public void deleteBrandById(Long[] ids);
	
	/**
	 * Title:findPage
	 * Description: 根据条件分页查询商品品牌
	 * @param brand	条件
	 * @param pageNum
	 * @param pageSize
	 * @return
	 */
	public PageResult findPage(TbBrand brand, int pageNum, int pageSize);
	
	/**
	 * Title:selectOptionList
	 * Description: 返回下拉列表
	 * @return
	 */
	public List<Map> selectOptionList();
}
